### PR TITLE
Add cron to run preflight against unstable image

### PR DESCRIPTION
**Pull request by sebrandon1**
Similar to: https://github.com/test-network-function/cnf-certification-test-partner/pull/368

### CODE REVIEW

File: .github/workflows/preflight.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        run: |
+          make build
+          ./preflight check container ${{ env.IMAGE_NAME }} --artifacts ${GITHUB_WORKSPACE}/openshift-preflight/artifacts
+
+          ALLOWED_FAILURES=("HasLicense" "RunAsNonRoot")
+          FAILURES=$(cat ${GITHUB_WORKSPACE}/openshift-preflight/artifacts/amd64/results.json | jq -r '.results.failed[].name')
+
+          for FAILURE in $FAILURES; do
+            FAILURE=$(echo $FAILURE | tr -d '"')
+            if [[ ! " ${ALLOWED_FAILURES[@]} " =~ " ${FAILURE} " ]]; then
```
Comment (rdavid) [inline .github/workflows/preflight.yml:45]: There is no need for the trailing spaces after and before ".

File: .github/workflows/preflight.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+              echo "ERROR: $FAILURE is not an allowed failure"
```
Comment (rdavid) [inline .github/workflows/preflight.yml:46]: Put errors to stderr, 'echo >&2 ...'

**Review by rdavid — APPROVED** [review]

File: .github/workflows/preflight.yml (hunk 1 above)
Comment (sebrandon1) [reply .github/workflows/preflight.yml]: Done.

File: .github/workflows/preflight.yml (hunk 2 above)
Comment (sebrandon1) [reply .github/workflows/preflight.yml]: Done.

**Review by jmontesi — APPROVED** [review]

**Pull request merged**